lang/en.yml

```
@@ -88,6 +88,8 @@ en:
     SAVEDRAFT: Save
     SEARCHRESULTS: 'Search results'
     SHOW_AS_LIST: 'show as list'
+    TOGGLE_CHILD_PAGES: 'Toggle child pages of {title}'
+    TOGGLE_CHILD_PAGES_ROOT: 'Toggle child pages of top level'
```

```suggestion
    TOGGLE_CHILD_PAGES_ROOT: 'Toggle site tree'
```
This probably makes more sense. The "child pages of top level" are really just the root level pages in the site tree - so you're toggling the whole tree.

---

Updated